<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sup</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>86</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="evolveButton">
         <property name="text">
          <string>Evolve Ant</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="visualizeButton">
         <property name="text">
          <string>Vizualize</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>evolveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onEvolveButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>visualizeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onVisualizeButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>756</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onEvolveButton()</slot>
  <slot>onVisualizeButton()</slot>
 </slots>
</ui>
